### PR TITLE
Bootstrap User profile page (#563)

### PR DESCRIPTION
**Pull request by bhavi**

### CODE REVIEW

File: app/views/devise/registrations/_password_details.html.erb [hunk 1]
```diff
@@ -1,44 +1,55 @@
...
+      <%= f.password_field(:password,  class: 'form-control', "aria-required": true , id: 'user_new_password') %>
     </div>
+  </div>
 
-    <div class="form-input">
-      <%= f.label(:password, _('New password'), class: 'required') %>
-      <%= f.password_field(:password, class: 'left-indent required input-medium', id: 'user_new_password') %>
-      <span role="" id="password_error"
-            class="error-tooltip-right left-indent"></span>
+  <div class="form-group">
```
Comment (briri) [inline app/views/devise/registrations/_password_details.html.erb]: Copy paste issue here? Don't think you meant to include the current password twice on this form

File: app/views/devise/registrations/_password_details.html.erb [hunk 2]
```diff
@@ -1,44 +1,55 @@
...
     </div>
-    
-    <br />
-    <div class='form-input'>
-      <div class="button-spacer">&nbsp;</div>
+  </div>
+
+  <div class="form-group">
+    <div class="col-sm-offset-2 col-sm-10">
       <%= render partial: 'shared/accessible_submit_button',
```
Comment (briri) [inline app/views/devise/registrations/_password_details.html.erb]: We're no longer using the accessible_submit_button, so go ahead and switch this to follow the button on another Bootstrapped page like `<%= f.button(_('Save'), class: "btn btn-default", type: "submit") %>`

File: app/views/devise/registrations/_personal_details.html.erb [hunk 3]
```diff
@@ -1,121 +1,128 @@
...
+  </p>
 
-  <div class="profile_text">
-    <p>
-      <%= _("Please note that your email address is used as your username.
-      If you change this, remember to use your new email address on sign in.") %>
-    </p>
-  </div>
+  <!-- Edit Profile -->
+  <%= _('You can edit any of the details below.') %>
```
Comment (briri) [inline app/views/devise/registrations/_personal_details.html.erb]: should be wrapped in a `<p>` or `<h2>` tag

File: app/views/devise/registrations/_personal_details.html.erb [hunk 4]
```diff
@@ -1,121 +1,128 @@
...
+      <div class="col-sm-6"><%= @user.api_token %></div>
+      <%= label_tag(:api_information, _('API Information'), class: "col-sm-2 control-label") %>
+      <div><%= link_to( _('How to use the API'), controller: "token_permission_types", action: "index")%></div>
+    </div>
+  <% end %>
 
-    </fieldset>
+  <div class="form-group">
+    <div class="col-sm-offset-2 col-sm-10">
+      <%= render partial: 'shared/accessible_submit_button',
```
Comment (briri) [inline app/views/devise/registrations/_personal_details.html.erb]: See above comment about using a standard Bootstrap button.

File: app/views/devise/registrations/_password_details.html.erb (hunk 1 above)
Comment (bhavi) [reply app/views/devise/registrations/_password_details.html.erb]: i think this happened by mistake during merge conflicts. fixed.

**Pull request merged**